doc/man3/EVP_set_default_properties.pod

```diff
@@ -3,14 +3,15 @@
 =head1 NAME
 
 EVP_set_default_properties, EVP_default_properties_enable_fips,
-EVP_default_properties_is_fips_enabled
-- Set default properties for future algorithm fetches
+EVP_default_properties_is_fips_enabled, EVP_get1_default_properties
```

Comment:
Should this be added to the NOTES section also?

Reply:
Fair point, will do